test/sample-test.ts

```diff
@@ -16,46 +51,54 @@ describe("Starknet", function () {
     const contract: StarknetContract = await contractFactory.deploy();
     console.log("Deployed at", contract.address);
     preservedAddress = contract.address;
-    await contract.invoke("increase_balance", [1, 20]);
-    console.log("Increased user 1 amount by 20");
-    const balanceStr1 = await contract.call("get_balance", [1]);
-    const balance1 = parseInt(balanceStr1);
-    expect(balance1).to.equal(20);
-
-    await contract.invoke("increase_balance", [2, 20]);
-    console.log("Increased user 2 amount by 20");
-    const balanceStr2 = await contract.call("get_balance", [2]);
-    const balance2 = parseInt(balanceStr2);
-    expect(balance2).to.equal(20);
+    
+    var r = "0x" + balance_signature_bettor.r.toString(16);
+    var s = "0x" + balance_signature_bettor.s.toString(16);
+    await contract.invoke("increase_balance", {user_id:BigInt(bettor_pub_key), amount:balance_amount},[BigInt(r),BigInt(s)]);
+    console.log("Increased user 1 amount by " + balance_amount);
+    const balanceStr1 = await contract.call("get_balance", {user_id:BigInt(bettor_pub_key)});
+    const balance1 = parseInt(balanceStr1.res);
```

Comment:
No need for parsing the output. In fact parsing is dangerous in case of very large output (precision issues with Cairo `felt` and JS `Number`). E.g. [here](https://github.com/Shard-Labs/starknet-hardhat-plugin#test) check how assertions/expectations are made directly with `BigInt` (no parsing).